### DISCUSSION TITLE
What happened with the real examples of Fault 8?

### DISCUSSION THREAD

**Discussion by FelipoDRS** [petrobras/3W#54] [Q&A] [answered]
Sorry if I missed some discussion or commit comment somewhere but why were they removed?

**Comment by utkuaslan85** [comment]
I was about to write same thing. I will use dataset for my masters thesis and planned to use real examples.

**Reply by ricardoevvargas** [reply]
Hi, @utkuaslan85.

It is very gratifying to know that 3W has arrived in Turkey and that it will be useful in your master's research. Suggestions for adjustments or additions to 3W that may make it more useful for academic research (whether yours or others) are welcome.

**Comment by ricardoevvargas** [comment, marked as answer]
Hi, @FelipoDRS. 

Thank you for asking this question as it allows clarification that may be useful to other 3W users and contributors.

You haven't missed any discussions or comments about it. Actual instances of type 8 were actually excluded from the 3W dataset in its version 1.1.0. Unfortunately this exclusion was not highlighted in the [release notes](https://github.com/petrobras/3W/blob/main/dataset/README.md#release-notes). An update will be made to these notes.

It is important to emphasize that it is expected that in new versions of the 3W dataset there will be exclusions (for different reasons) and also inclusions of instances. Between the releases of versions 1.0.0 and 1.1.0 there was a considerable hiatus, but as this project is currently more structured, we expect that new versions will be released with smaller intervals (even if not periodically or previously defined). We are working to release the next versions with a considerably higher number of instances. More types of events will also be gradually added.

**Reply by utkuaslan85** [reply]
Hi, Ricardo.
Thanks for the information. If it is OK, can you give further information why there were removed from dataset? I was planning to use real instances from Fault 8 for my project. Now I am in doubt whether to use them or not. Should I use them?
Thank you.

**Reply by ricardoevvargas** [reply]
Hi, @utkuaslan85.

The type 8 instances that were removed can be used in general analyses. However, the default suggestion will always be to use the latest version of 3W resources, which will be updated gradually.

**Reply by utkuaslan85** [reply]
Hi, Ricardo.
Thank you for your reply. Is there a roughly roadmap how will dataset will evolve? I mean something showing when approximately new data will be added, how many instances, to which event, from which type of source etc.

**Reply by ricardoevvargas** [reply]
We don't have a roadmap like you described yet, but we have a [backlog](https://github.com/petrobras/3W/blob/main/BACKLOG.md) (list of priority improvements for the 3W project that we intend to develop collaboratively with the community). In [this guide](https://github.com/petrobras/3W/blob/main/CONTRIBUTING.md) we present how each type of contributions that we expect can be proposed.
